src/common/transformations/src/transformations/common_optimizations/reshape_sequence_fusion.cpp

```diff
@@ -17,21 +17,53 @@
 NGRAPH_RTTI_DEFINITION(ngraph::pass::ReshapeSequenceFusion, "ReshapeSequenceFusion", 0);
 
 namespace {
+OPENVINO_SUPPRESS_DEPRECATED_START
 bool has_valid_pattern(const std::shared_ptr<ngraph::Node> & node) {
     auto const_node = std::dynamic_pointer_cast<ngraph::opset8::Constant>(node);
-    if (!const_node) return false;
+    if (!const_node) {
+        // Lower bound of the value
+        ngraph::HostTensorPtr lb = std::make_shared<ngraph::HostTensor>(node->output(0));
```

Comment:
I think it is better to work with particular output port which is consumed by reshape instead of relying on zero index.

Reply:
I've changed the `is_valid_pattern` helper to take `Output<Node>` as parameter, does it ok?